README.md

```diff
@@ -1,19 +1,44 @@
-# App ID module
-<!-- UPDATE BADGE: Update the link for the following badge-->
-[![Incubating (Not yet consumable)](https://img.shields.io/badge/status-Incubating%20(Not%20yet%20consumable)-red)](https://terraform-ibm-modules.github.io/documentation/#/badge-status)
+<!-- Update the title -->
+# IBM AppID module
```

Comment:
Perhaps remove the words "module" from it as the repo will eventually contain more than just the module (will contain DA too). Maybe just go with "App ID"